site-templates/pages/create-canvas-and-add-text.gomd

```diff
@@ -346,19 +356,26 @@ From this response, you can obtain the newly created Canvas ID: ['data']['id']
         },
         "traits": null,
         "pinned": false,
-        "type": "Note",
-        "width": 560,
-        "height": 320,
-        "text": "Note tests",
+        "type": "Text",
+        "text": "Text Example",
         "style": {
-            "hidden": false,
-            "fontSize": 64,
+            "width": 256,
+            "height": 298,
             "fontWeight": "normal",
             "textTransform": "none",
+            "fontSize": 64,
+            "fontFamily": "Dosis",
+            "fontStyle": "normal",
+            "color": {
+                "r": 234,
+                "g": 62,
+                "b": 53,
+                "a": 1
+            },
             "backgroundColor": {
-                "r": 56,
-                "g": 138,
-                "b": 135,
+                "r": 0,
+                "g": 0,
+                "b": 0,
                 "a": 1
```

Comment:
@Codiegladney-bluescape : Change this value to "0". Alpha channel "0" means transparent. Currently the transparent background does not work for Text (shows black), but let's set it correctly for when the bug is fixed.

Reply:
Sorry, this is the response....